environment/environment_templates.go

```diff
@@ -279,9 +293,11 @@ func NewMixedVersionChainlinkCluster(nodeCount, pastVersionsCount int) K8sEnvSpe
 		chainlinkGroup.manifests = append(chainlinkGroup.manifests, cManifest)
 	}
 
+	dependencyGroups := []*K8sManifestGroup{}
 	dependencyGroup := getBasicDependencyGroup()
 	addPostgresDbsToDependencyGroup(dependencyGroup, nodeCount)
-	return addNetworkManifestToDependencyGroup("mixed-version-chainlink", dependencyGroup, chainlinkGroup)
+	dependencyGroups = append(dependencyGroups, dependencyGroup)
```

Comment:
Another one that can be simplified: `dependencyGroups := []*K8sManifestGroup{dependencyGroup}`